dist/DropDown.js

```diff
@@ -33,7 +33,7 @@ const DropDown = forwardRef((props, ref) => {
           {list.map((_item, _index) => (<Menu.Item key={_index} theme={theme} titleStyle={{
         color: value === _item.value
             ? activeColor || (theme || activeTheme).colors.primary
-            : undefined,
+            : ((theme || activeTheme) ? (theme || activeTheme).colors.text : undefined),
```

Comment:
This will also do the job
 titleStyle={value === _item.value? {color:  activeColor || (theme || activeTheme).colors.primary} : {}}
 
 This way it'll just use the global theme's colors if not the active one

Reply:
Very good solution. Now, to see if the developer of the component accepts the PR.